doc/src/overrides.md

```diff
@@ -83,13 +83,16 @@ a directory, the latter is used for backwards compatibility. The files use the
 ``` toml
 [toolchain]
 channel = "nightly-2020-07-10"
+# or
+path = "/path/to/local/toolchain"
+
 components = [ "rustfmt", "rustc-dev" ]
 targets = [ "wasm32-unknown-unknown", "thumbv2-none-eabi" ]
 profile = "minimal"
```

Comment:
I think the `#or ...` bit should be a separate example because for paths, components, targets, and profile will have no meaning whatsoever.